core/controllers/contributor_dashboard.py

```diff
@@ -276,24 +284,25 @@ def _get_state_names_to_not_in_review_content_id_mapping(
         minus any contents found in suggestions.
 
         Args:
-            state_names_to_content_id_mapping: dict(str, dict(str, str)). A dict
-                where state_name is the key and a dict with content_id as the
-                key and html content as value.
+            state_names_to_content_id_mapping: dict(str, dict(str, WrittenTranslation)). A dict # pylint: disable=line-too-long
```

Comment:
It is 88 characters before `A dict`. Also, the linter seems to be a bit off here because I tried the following and it gave me an error. The only workaround I could come up with is what I have here.

This is what I tried earlier:
```py
    def _get_state_names_to_not_in_review_content_id_mapping(
            self, state_names_to_content_id_mapping, suggestions):
        """Returns a copy of the supplied state_names_to_content_id_mapping
        minus any contents found in suggestions.

        Args:
            state_names_to_content_id_mapping: dict(str, dict(str,
                WrittenTranslation)). A dict where state_name is the key and a
                dict with content_id as the key and WrittenTranslation as value.
            suggestions: list(Suggestion). A list of translation suggestions.

        Returns:
            dict(str, dict(str, WrittenTranslation)). A dict where state_name is
            the key and a dict with content_id as the key and WrittenTranslation
            as value.
        """
```

and this is the error I got:
```
----------------------------------------
Please fix the errors below:
----------------------------------------
************* Module oppia.core.controllers.contributor_dashboard
W:281, 4: "state_names_to_content_id_mapping" missing in parameter documentation 
W:281, 4: "state_names_to_content_id_mapping" missing in parameter type documentation 

---------------------------
Checks Not Passed.
---------------------------
```

Reply:
Ah sorry, I meant we could keep the lint disable statement but indent after `dict(str, dict(str, WrittenTranslation)).` as that's the portion we want to keep on one line.

Reply:
Weird, that also throws an error:

```
----------------------------------------
Please fix the errors below:
----------------------------------------
************* Module oppia.core.controllers.contributor_dashboard
W:281, 4: Arguments should be in following form: variable_name: typeinfo. Description. 

---------------------------
Checks Not Passed.
---------------------------
```

Reply:
Ah, I see. Didn't realize how strict our lint checks were. Never mind then! I wonder if we should file a ticket to relax the lint requirements somewhat for cases like these. I've also had issues passing the lint checks for various not-so-clear reasons.

Reply:
Yep, makes sense. I've created https://github.com/oppia/oppia/issues/13411 to track it.